src/Vendor/Laravel/Controllers/Stats.php

```diff
@@ -119,7 +119,7 @@ public function apiLog($uuid)
                             'is_json',
                             'wants_json',
                             'error_id',
-                            'updated_at',
```

Comment:
Session record is updated frequently, so created_at will have a different timestamp, but we could add created_at here